content/blog/2018-08-14-where-to-bird.md

```diff
@@ -0,0 +1,365 @@
+    ##         $overpass_call : The call submitted to the overpass API
+    ##             $timestamp : [ Thu 5 Jul 2018 08:06:35 ]
+    ##            $osm_points : 'sf' Simple Features Collection with 1 points
+    ##             $osm_lines : 'sf' Simple Features Collection with 0 linestrings
+    ##          $osm_polygons : 'sf' Simple Features Collection with 0 polygons
+    ##        $osm_multilines : 'sf' Simple Features Collection with 0 multilinestrings
+    ##     $osm_multipolygons : 'sf' Simple Features Collection with 0 multipolygons
+
+``` r
+results$osm_points
+```
+
+    ##              leisure            geometry
+    ## 5004940425 bird_hide 8.920901, 47.741569
+
+Yay, we now know where to find a bird hide not too far from the MPI!
+
+#### Visualizing our location and the bird hide
+
+So one could enter the coordinates of that bird hide in one’s favourite
+mapping software or app but to show you where the bird hide is we can
+actually step back and use
+[`osmplotr`](https://github.com/ropensci/osmplotr), another package
+contributed to rOpenSci by Mark Padgham!
+
+The way `osmplotr` works is letting you create a basemap, on which
+you’ll add different layers extracted from OpenStreetMap using
+`osmplotr::extract_osm_objects` or `osmdata` functions directly. Its
+strengths are therefore the use of open data, and the customization of
+what you’re using as background!
+
+Let’s create a basemap for our bounding box, and then add roads and
+buildings to it.
+
+``` r
+library("osmplotr")
+```
+
+    ## Data (c) OpenStreetMap contributors, ODbL 1.0. http://www.openstreetmap.org/copyright
+
+``` r
+bbox <- get_bbox(bbox)
+dat_B <- extract_osm_objects (key = 'building', bbox = bbox)
+dat_H <- extract_osm_objects (key = 'highway', bbox = bbox)
+
+map0 <- osm_basemap(bbox = bbox, bg = 'gray20') %>%
+  add_osm_objects (dat_B, col = 'gray40') %>%
+  add_osm_objects (dat_H, col = 'gray80')
+```
+
+``` r
+map0 %>%
+  add_axes() %>%
+  print_osm_map (filename = 'map_a1.png', width = 600,
+               units = 'px', dpi = 72)
+
+library("magrittr")
+magick::image_read('map_a1.png') %>%
+  magick::image_annotate("Map data © OpenStreetMap contributors",
+                         color = "white",
+                         boxcolor =  "black",
+                         size = 15,
+                         gravity = "south")
+```
+
+<img src="/img/blog-images/2018-08-14-where-to-bird/map1-print-1.png" width="600" />
+
+Quite pretty! The lakes can be seen because of the absence of roads and
+buildings on them.
+
+Now, let’s plot the bird hide and the MPI on them. Since we used
+`osmdata::osmdata_sf`, we had gotten the data in a receivable class,
+`sf`.
+
+``` r
+points_map <- add_osm_objects(map0, results$osm_points, 
+                              col = 'salmon',
+                              size = 5)
+```
+
+For plotting the MPI, we’ll convert `opencage` output to an `sf` point
+with the same coordinate reference system as the Openstreetmap data
```

Comment:
Openstreetmap vs OpenStreetMap elsewhere

Reply:
Thanks!

Reply:
Sorry @maelle, I've probably confused this issue because I write OpenStreetMap, but technically it is just plain English "Open Street Map".